core/interfaces/branch.py

```diff
@@ -58,7 +58,7 @@ class BranchInterface(ui.Interface, GitCommand):
       [H] diff history against active
       [e] toggle display of remote branches
 
-      [tab] transition to rebase dashboard
+      [tab] transition to rebase dashboard          [E] edit branch description
```

Comment:
Let's rearrange a little bit.  Can you update the entire section to match following?
```
      #############
      ## ACTIONS ##
      #############

      [c] checkout                                  [p] push selected to remote
      [b] create new branch (from HEAD)             [P] push all branches to remote
      [d] delete                                    [D] delete (force)
      [R] rename (local)                            [m] merge selected into active branch
      [t] configure tracking                        [M] fetch and merge into active branch
      [o] checkout remote as local                  [h] fetch remote branches

      [f] diff against active
      [H] diff history against active
      [E] edit branch description

      [e]   toggle display of remote branches
      [tab] transition to rebase dashboard
      [r]   refresh
```